src/components/Dashboard/index.js

```diff
@@ -327,14 +328,25 @@ class Dashboard extends Component {
 
   render() {
     const {
-      hasWallet, collateralToken, accountPredictiveAssets,
+      hasWallet, collateralToken, accountPredictiveAssets, userTournamentInfo,
     } = this.props
 
     let metricsSection = <div />
     let tradesHoldingsSection = <div className="dashboardWidgets dashboardWidgets--financial" />
-    const predictedProfitFormatted = Decimal(accountPredictiveAssets).toDP(4, 1).toString()
+    const predictedProfitFormatted = Decimal(accountPredictiveAssets)
```

Comment:
Please use the function in `DecimalValue` called `decimalToText` to format this string, so it's consistent everywhere